roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -80,31 +80,43 @@ spec:
       priorityClassName: '{{ control_plane_priority_class }}'
 {% endif %}
       initContainers:
-{% if bundle_ca_crt or projects_persistence|bool or init_container_extra_commands %}
-        - name: init
+{% if bundle_ca_crt  %}
+        - name: init-bundle-ca-trust
           image: '{{ _init_container_image }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           resources: {{ init_container_resource_requirements }}
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
```

Comment:
@JoelKle This will break on Openshift.  Can we template this in only if is_k8s=true?

Reply:
Did `update-ca-trust extract` require more permissions than just `update-ca-trust`?  I am trying to understand why the securityContext bit was added.

Reply:
I removed the whole `securityContext` block from this PR. Now the init-container behaves as before.

TL;DR
Withouth the `runAsUser: 0` parameter the init-container runs the update-ca-trust command as `uid=1000(awx) gid=0(root)`. That means the files in the folder `/etc/pki/ca-trust/extracted` are owned by `awx:root`
With the `runAsUser: 0` parameter all files are owned by `root:root` which is more accurate.

Reply:
The `update-ca-trust extract` behaves the same as withouth the `extract`. However with the `extract` the command may print more warnings, ...
Check the manpage of update-ca-trust extract > "COMMANDS"